packages/pigment-css-react/src/Box.jsx

```diff
@@ -51,4 +52,394 @@ const Box = React.forwardRef(
+      'body',
+      'br',
+      'button',
+      'canvas',
+      'caption',
+      'center',
+      'circle',
+      'cite',
+      'clipPath',
+      'code',
+      'col',
+      'colgroup',
+      'data',
+      'datalist',
+      'dd',
+      'defs',
+      'del',
+      'desc',
+      'details',
+      'dfn',
+      'dialog',
+      'div',
+      'dl',
+      'dt',
+      'ellipse',
+      'em',
+      'embed',
+      'feBlend',
+      'feColorMatrix',
+      'feComponentTransfer',
+      'feComposite',
+      'feConvolveMatrix',
+      'feDiffuseLighting',
+      'feDisplacementMap',
+      'feDistantLight',
+      'feDropShadow',
+      'feFlood',
+      'feFuncA',
+      'feFuncB',
+      'feFuncG',
+      'feFuncR',
+      'feGaussianBlur',
+      'feImage',
+      'feMerge',
+      'feMergeNode',
+      'feMorphology',
+      'feOffset',
+      'fePointLight',
+      'feSpecularLighting',
+      'feSpotLight',
+      'feTile',
+      'feTurbulence',
+      'fieldset',
+      'figcaption',
+      'figure',
+      'filter',
+      'footer',
+      'foreignObject',
+      'form',
+      'g',
+      'h1',
+      'h2',
+      'h3',
+      'h4',
+      'h5',
+      'h6',
+      'head',
+      'header',
+      'hgroup',
+      'hr',
+      'html',
+      'i',
+      'iframe',
+      'image',
+      'img',
+      'input',
+      'ins',
+      'kbd',
+      'keygen',
+      'label',
+      'legend',
+      'li',
+      'line',
+      'linearGradient',
+      'link',
+      'main',
+      'map',
+      'mark',
+      'marker',
+      'mask',
+      'menu',
+      'menuitem',
+      'meta',
+      'metadata',
+      'meter',
+      'mpath',
+      'nav',
+      'noindex',
+      'noscript',
+      'object',
+      'ol',
+      'optgroup',
+      'option',
+      'output',
+      'p',
+      'param',
+      'path',
+      'pattern',
+      'picture',
+      'polygon',
+      'polyline',
+      'pre',
+      'progress',
+      'q',
+      'radialGradient',
+      'rect',
+      'rp',
+      'rt',
+      'ruby',
+      's',
+      'samp',
+      'script',
+      'search',
+      'section',
+      'select',
+      'set',
+      'slot',
+      'small',
+      'source',
+      'span',
+      'stop',
+      'strong',
+      'style',
+      'sub',
+      'summary',
+      'sup',
+      'svg',
+      'switch',
+      'symbol',
+      'table',
+      'tbody',
+      'td',
+      'template',
+      'text',
+      'textarea',
+      'textPath',
+      'tfoot',
+      'th',
+      'thead',
+      'time',
+      'title',
+      'tr',
+      'track',
+      'tspan',
+      'u',
+      'ul',
+      'use',
+      'var',
+      'video',
+      'view',
+      'wbr',
+      'webview',
+    ]),
+    PropTypes.func,
+  ]),
+  /**
+   * The content of the component.
+   */
+  children: PropTypes.node,
+  /**
+   * The component used for the root node.
+   * Either a string to use a HTML element or a component.
+   */
+  component: PropTypes.oneOfType([
```

Comment:
@michaldudak do you remember what we were doing for this to resolve to only `PropTypes.elementType`?

Reply:
I fixed this in https://github.com/mui/pigment-css/pull/16/commits/ae50f99577e0b1e278160532244898d543ccafdb, I couldn't figure out a better way.

Reply:
Your solution looks fine. I don't know why it doesn't work like in Core, though.